src/components/Speciality/Tutorials.js

```diff
@@ -1,55 +1,37 @@
 import React from 'react'
-import styled from 'styled-components'
-import remcalc from 'remcalc'
 import { Padding } from 'styled-components-spacing'
 
-import { Row, Col, Grid } from '../grid'
 import GreyBackground from '../Common/GreyBackground'
-import { SectionTitle, Subtitle, BodyPrimary } from '../Typography'
-import ExternalAnchor from '../Common/ExternalAnchor'
-import StyledLink from '../Common/StyledLink'
-import Hr from '../Common/Hr'
+import { Grid } from '../grid'
+import TitleAndMediaList from '../Common/TitleAndMediaList'
 
-const TutorialsGrid = styled(Grid)`
-  padding-bottom: ${remcalc(18)};
-`
+const TutorialsSection = ({ tutorials }) => {
+  const mediaItems = tutorials
+    .slice(0, 3)
+    .map(({ id, title, link, additionalInfo }) => {
+      return {
+        id,
+        title,
+        href: link,
+        body: additionalInfo
+      }
+    })
 
-const TutorialsSection = ({ tutorials, externalResources }) =>
-  tutorials.length > 0 ? (
+  return (
     <GreyBackground>
-      <TutorialsGrid>
-        <Padding vertical={5}>
-          <Row>
-            <Col width={[1, 1, 1, 1, 6 / 12]}>
-              <SectionTitle>Tutorials</SectionTitle>
-              <BodyPrimary>
-                NodeJS tutorials created by members of YLD for the community.
-              </BodyPrimary>
-            </Col>
-            <Col width={[1, 1, 1, 1, 4 / 12]}>
-              <ul>
-                {tutorials.slice(0, 3).map(externalResource => (
-                  <li key={`${externalResource.id}`}>
-                    <Subtitle noPaddingBottom>
-                      <ExternalAnchor href={externalResource.link}>
-                        {externalResource.title}
-                      </ExternalAnchor>
-                    </Subtitle>
-                    {externalResource.additionalInfo}
-                    <Hr />
-                  </li>
-                ))}
-              </ul>
-              <Padding top={3}>
-                <StyledLink external href="http://nodetuts.com/">
-                  More tutorials
-                </StyledLink>
-              </Padding>
-            </Col>
-          </Row>
+      <Grid>
+        <Padding vertical={{ desktop: 5, smallPhone: 3.5 }}>
+          <TitleAndMediaList
+            title="Tutorials"
+            description="NodeJS tutorials created by members of YLD for the community."
+            mediaItems={mediaItems}
+            CTALink="http://nodetuts.com/"
+            CTAText="More tutorials"
```

Comment:
not a requested change but hard-coded strings/descriptions should come from contentful.

Reply:
Yeah I completely agree that these should come from Contentful. But I would suggest that we consider alterations to Contentful to be out of the scope of this ticket.

If it were just a matter of adding a couple of new fields, then we could definitely do this at the same time :+1:

But we don't currently have a content type on Contentful, which is for entering information which is to be used across all Speciality pages. Things that would fall under this category include the CTALink & CTAText for this blog section, but also the section titles & section descriptions for many of the other sections on these pages.

Right now, most of these things are hard-coded into the template. The only things in Contentful are the page-specific things (i.e. the ones that change depending on whether it's a React page or a Node page).

So, in order to do what you're suggesting, we would ideally be creating a new content type for the Speciality _Template_ itself - things to be shared across all speciality pages. Rather than creating a whole new content type right now, just for these 2 strings, I would suggest that we wait until we can do it in a more systematic and thorough way, that accounts for all of the common elements - basically when Ollie joins us.

Reply:
That makes sense. Let's talk about it when Ollie is with us 👍